python/housinginsights/sources/census.py

```diff
@@ -1,7 +1,7 @@
 import sys, os
```

Comment:
This file should be reverted - see my comments on this in the main discussion thread coming in just a minute.